scripts/release/package_audit.rb

```diff
@@ -0,0 +1,233 @@
+version_tag = ARGV[0] || "latest"
+
+# Manual list - MAKE SURE UP TO DATE especially base images
+containers = [
+  { name: "ballaerospace/cosmosc2-ruby:#{version_tag}", base_image: "alpine:3.15.0", apk: true, gems: true },
+  { name: "ballaerospace/cosmosc2-node:#{version_tag}", base_image: "ballaerospace/cosmosc2-ruby:#{version_tag}", apk: true },
+  { name: "ballaerospace/cosmosc2-base:#{version_tag}", base_image: "ballaerospace/cosmosc2-ruby:#{version_tag}", apk: true, gems: true },
+  { name: "ballaerospace/cosmosc2-cmd-tlm-api:#{version_tag}", base_image: "ballaerospace/cosmosc2-base:#{version_tag}", apk: true, gems: true },
+  { name: "ballaerospace/cosmosc2-init:#{version_tag}", base_image: "ballaerospace/cosmosc2-base:#{version_tag}", apk: true, gems: true, yarn: "/cosmos/plugins/yarn.lock" },
+  { name: "ballaerospace/cosmosc2-operator:#{version_tag}", base_image: "ballaerospace/cosmosc2-base:#{version_tag}", apk: true, gems: true },
+  { name: "ballaerospace/cosmosc2-script-runner-api:#{version_tag}", base_image: "ballaerospace/cosmosc2-base:#{version_tag}", apk: true, gems: true },
+  { name: "ballaerospace/cosmosc2-redis:#{version_tag}", base_image: "redis:6.2", apt: true },
+  { name: "ballaerospace/cosmosc2-traefik:#{version_tag}", base_image: "traefik:2.4.13", apk: true },
+  { name: "ballaerospace/cosmosc2-minio-init:#{version_tag}", base_image: "minio/mc:RELEASE.2021-12-10T00-14-28Z", rpm: true },
+  { name: "minio/minio:RELEASE.2021-06-17T00-10-46Z", rpm: true },
```

Comment:
For the previous 4 lines you're relying on a specific version. Ultimately I'd love to consolidate this but we'll just have to remember to update when we change the dockerfiles.